breathe-and-relax-app/style.css

```diff
@@ -87,29 +87,29 @@ body {
   }
   
 .container.grow {
-    animation: grow 3s linear forwards;
-  }
-  
-  @keyframes grow {
-    from {
-      transform: scale(1);
-    }
+  animation: grow 3s ease-out forwards;
```

Comment:
I changed the animation to use ease-out instead of linear as I thought it looked nicer